src/components/filterdialog/filterdialog.template.html

```diff
@@ -140,4 +140,9 @@
         <div class="collapseContent filterOptions">
         </div>
     </div>
+
+    <button is="emby-button" type="button" class="btnResetAllFilters hide emby-button raised">
```

Comment:
```suggestion
    <button is="emby-button" type="button" class="btnResetAllFilters hide emby-button raised block">
```
I think it should take up full width.